### ISSUE TITLE
Programatically update buttons

### ISSUE THREAD

**Issue by CBenni** [noahm/featured-follow-button#1]
Is there a way to update the buttons programmatically, say, through an API?
From what I can gather via the README, the backend is no longer used (I assume thanks to the new twitch extension datastore?) - can 3rd party apps interact with that by any chance?
Thanks!

**Comment by noahm** [comment]
No, not yet. I'm not sure what the best way to approach that is. Do you
know if any other extensions that provide similar access?

**Comment by CBenni** [comment]
I mean, plenty of extensions have integrations with external services, usually in liaison with complex web applications however. I am not sure if there's anything that implements a "minimal" implementation of external API access.
I am not too well-accustomed to twitchs new data store API, but would it be possible to just expose the authentication keys used broadcaster side (in the configuration dialog, for example) and allowing 3rd party services to directly interact with storage for said channel?
Thanks!

**Comment by noahm** [comment]
No, that's not really possible. All access to the amazon data store is authenticated via JWT signatures which have a fixed expiration date.

I would have to re-introduce a backend specifically for this extension to provide a public API for this sort of use. The main question I have is how to handle authentication for those requests, and I'm afraid the only answer is implementing twitch OAuth login. (quite a bit of work)

**Comment by CBenni** [comment]
Ah, fair enough. Thanks for you response!

**Comment by noahm** [comment]
No worries, it's a great suggestion just the same and I'd love to add a public API eventually! I'll leave this open as the task to do so.
